event/eth_verify_redeem.go

```diff
@@ -71,19 +71,20 @@ func (job *JobETHVerifyRedeem) DoMyJob(ctx interface{}) {
 	}
 
 	addr := ethCtx.GetValidatorETHAddress()
-	status, err := cd.VerifyRedeem(addr, msg.From())
-	if err != nil {
-		ethCtx.Logger.Error("Error in verifying redeem :", job.GetJobID(), err)
+	status := cd.VerifyRedeem(addr, msg.From())
+	if status == ethereum.ErrorConnecting {
+		ethCtx.Logger.Error("Unable to connect to ethereum smartcontract")
+		return
 	}
-	if err == ethereum.ErrRedeemExpired {
+	if status == ethereum.Expired {
```

Comment:
Is this the only case where user gets his oETH refunded? Did u take care of other cases?